core-bundle/src/Controller/FrontendModule/TwoFactorController.php

```diff
@@ -84,6 +89,10 @@ protected function getResponse(Template $template, ModuleModel $model, Request $
             return new Response('', Response::HTTP_NO_CONTENT);
         }
 
+        if (!$this->get('security.helper')->isGranted('IS_AUTHENTICATED_FULLY')) {
```

Comment:
Why not use an annotation?

```php
/**
 * @IsGranted("IS_AUTHENTICATED_FULLY")
 */
```

Reply:
Didn't think about it.

Reply:
What's the advantages?

Reply:
It is less code and Symfony will automatically handle the request so we do not have to return a response.

Reply:
I'll adapt.

Reply:
The annoations requires the `FrameworkExtraBundle` and we don't want this bundle as a dependency.